<launch>
	<env name="GAZEBO_MODEL_PATH" value="$(find potbot_description)/models:$(optenv GAZEBO_MODEL_PATH)"/>
  <arg name="x_pos"             default="0.0"/>
  <arg name="y_pos"             default="0.0"/>
  <arg name="z_pos"             default="0.0"/>

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="paused" value="false"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="true"/>
    <arg name="headless" value="false"/>
    <arg name="debug" value="false"/>
  </include>

  <include file="$(find potbot)/launch/gazebo/spawn_model/spawn_turtlebot3.launch">
    <arg name="tf_prefix"     value="robot_0"/>
    <arg name="model"         value="waffle_pi"/>
    <arg name="x_pos"         value="$(arg x_pos)"/>
    <arg name="y_pos"         value="$(arg y_pos)"/>
    <arg name="z_pos"         value="$(arg z_pos)"/>
  </include>

</launch>
